cypress/e2e/with-users/machines/actions.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { generateMAASURL } from "../../utils";
+
+const MACHINE_ACTIONS = [
+  "Commission",
+  "Allocate",
+  "Deploy",
+  "Release",
+  "Abort",
+  "Clone from",
+  "Power On",
+  "Power off",
+  "Test",
+  "Enter rescue mode",
+  "Exit rescue mode",
+  "Mark fixed",
+  "Mark broken",
+  "Override failed testing",
+  "Lock",
+  "Unlock",
+  "Tag",
+  "Set zone",
+  "Set pool",
+  "Import images",
```

Comment:
As Adam said, this needs to be removed since it's not a valid action for a machine
```suggestion

```